🐛 Make `kubestellar init` support syncer updating status

MikeSpreitzer opened the pull request:
<!--
Thanks for creating a pull request!

If this is your first time, please make sure to review CONTRIBUTING.MD.

Please copy the appropriate `:text:` or icon to the beginning of your PR title:

:sparkles: ✨ feature
:bug: 🐛 bug fix
:book: 📖 docs
:memo: 📝 proposal
:warning: ⚠️ breaking change
:seedling: 🌱 other/misc
:question: ❓ requires manual review/categorization

-->
## Summary
This PR extends what `kubestellar init` does, making it also add the RBAC objects to the `root:compute` workspace that enable the syncer to update the `status` sub-objects of the objects whose Kind is defined by the kube APIExport from `root:compute`.

## Related issue(s)

Fixes #477 
Fixes #546

/cc @yana1205 
/cc @dumb0002

MikeSpreitzer commented:
This PR is currently a work-in-progress, although I fear to formally mark it as such for fear of suppressing automated checking.  I am waiting on answers to questions in #477 and testing whether the different YAML used here works.

clubanderson commented:
bug in gh action is fixed.  unfortunately there is no way to update the commit for the workflow file that is used by the action to use the fix.  You can continue with this PR and ignore that test when it is time to merge.  New PRs will not fail on this action.  Looks like dependabot is unable to access secrets when it does a PR.  This is a known issue.  So I hope that this does not happen again unless dependabot steps in again.

yana1205 commented on scripts/kubestellar:
```diff
@@ -86,7 +86,55 @@ then
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: kcp-admin
```
Could you please correct the subject to the following one? 
```
subjects:
- apiGroup: rbac.authorization.k8s.io
  kind: Group
  name: apis.kcp.io:binding:system:authenticated
```

MikeSpreitzer replied:
done

MikeSpreitzer commented:
The force-push to c4078ef6 is a rebase plus the change requested in review above.
I think this relatively broad permission is OK because it is used in just one of two tests that both have to Allow, the other one being the regular check in the mailbox workspace.

MikeSpreitzer commented:
@clubanderson : the rebase seems to have picked up the fixed GH Action.

MikeSpreitzer commented:
Tested successfully.

```console
(base) mspreitz@mjs12 edge-mc % kubectl ws .
Current workspace is "root:espw:c4xthhgidi9hnsx9-mb-f4015fdc-c2c6-405c-822f-a5e25755ba04".
(base) mspreitz@mjs12 edge-mc % 
(base) mspreitz@mjs12 edge-mc % kubectl get deployments -A
NAMESPACE      NAME       READY   UP-TO-DATE   AVAILABLE   AGE
commonstuff    commond    1/0     1            1           8m46s
specialstuff   speciald   1/0     1            1           8m43s
```

Note: those zeros are there because `spec.replicas` is unset.

MikeSpreitzer commented:
BTW, I discovered during testing that the verbosity control in the `kubestellar` script is broken, so I fixed that.  I also added a `-X` option and fixed up the options descriptions.

yana1205 commented:
/lgtm

Thanks!

MikeSpreitzer commented:
/approve
